9-type-aliases-redux/9-type-aliases-redux.md

```diff
@@ -19,6 +19,19 @@ variable identifiers, but this may change in the future. We restrict type
 variable identifiers from using keywords in the same way (to support 
 extraction).
 
+## Type Pattern
+
+Type patterns are the part of the language between `type` and `=` in `type a = <type> in e`
+
+It is represented as greek letter `rho` in the attached
+[latex document](./latex/kind-judgements.pdf).
+
+```
+type TPat.t =
+  | EmptyHole(MetaVar.t)
+  | TyVar(VarErrStatus.t, TyId.t)
```

Comment:
don't need a VarErrStatus here, since we're introducing a binding not consuming it